specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14/keyvault.json

```diff
@@ -0,0 +1,1359 @@
+{
```

Comment:
This files should be referenced on the readme.md file

Reply:
Added.